### PR TITLE
Adding ssie events for longevity

### PR DESCRIPTION
**Pull request by lsrinivas-pure**
**What this PR does / why we need it**:
Added below events for Longevity

1. Pool Maintenance mode
2. Node Maintenance mode
3. Kill Px Daemon
4. pool full scenarios
5. ha increase with pvc resize

**Which issue(s) this PR fixes** (optional)
Closes #PTX-21257

**Special notes for your reviewer**:

### CODE REVIEW

File: tests/testTriggers.go [hunk 1]
```diff
@@ -1088,6 +1100,186 @@ func TriggerHAIncrease(contexts *[]*scheduler.Context, recordChan *chan *EventRe
...
+		event.End = time.Now().Format(time.RFC1123)
+		*recordChan <- event
+	}()
+	setMetrics(*event)
+
+	expReplMap := make(map[*volume.Volume]int64)
+	stepLog := "get volumes for all apps in test and increase replication factor"
+	Step(stepLog, func() {
+		log.InfoD(stepLog)
+		time.Sleep(10 * time.Minute)
```
Comment (kphalgun-px) [inline tests/testTriggers.go:1127]: wait time necessary?

File: tests/testTriggers.go [hunk 2]
```diff
@@ -1088,6 +1100,186 @@ func TriggerHAIncrease(contexts *[]*scheduler.Context, recordChan *chan *EventRe
...
+						}
+
+						err = volumeResize(v)
+						UpdateOutcome(event, err)
+
+					})
+				stepLog = fmt.Sprintf("validate successful repl increase on app %s's volume: %v",
+					ctx.App.Key, v)
+				Step(stepLog,
+					func() {
```
Comment (kphalgun-px) [inline tests/testTriggers.go:1245]: log.InfoD?

File: tests/testTriggers.go [hunk 3]
```diff
@@ -1375,6 +1627,308 @@ func TriggerRestartVolDriver(contexts *[]*scheduler.Context, recordChan *chan *E
...
+			for poolUUID, v := range poolsStatus {
+				if v == "Offline" {
+					taskStep := fmt.Sprintf("expanding storagefull pool [%s] on node [%s]", poolUUID,
+						appNode.Name)
+					event.Event.Type += "<br>" + taskStep
+					log.InfoD("Pool [%s] on node [%s] is offline", v, appNode.Name)
+					poolResizeType := []opsapi.SdkStoragePool_ResizeOperationType{opsapi.SdkStoragePool_RESIZE_TYPE_AUTO,
+						opsapi.SdkStoragePool_RESIZE_TYPE_RESIZE_DISK}
+
+					//Remove it once SdkStoragePool_RESIZE_TYPE_ADD_DISK is supported in dmthin
```
Comment (kphalgun-px) [inline tests/testTriggers.go:1842]: // TODO: Remove it once SdkStoragePool_RESIZE_TYPE_ADD_DISK is supported in dmthin?

File: tests/testTriggers.go [hunk 4]
```diff
@@ -1375,6 +1627,308 @@ func TriggerRestartVolDriver(contexts *[]*scheduler.Context, recordChan *chan *E
...
+							UpdateOutcome(event, err)
+							return
+						}
+					})
+					stepLog = fmt.Sprintf("Ensure that pool %s expansion is successful", selectedPool.Uuid)
+					Step(stepLog, func() {
+						log.InfoD(stepLog)
+
+						err = waitForPoolToBeResized(expandedExpectedPoolSize, selectedPool.Uuid)
+						log.FailOnError(err, "Error waiting for poor resize")
```
Comment (kphalgun-px) [inline tests/testTriggers.go:1888]: UpdateOutcome?

File: tests/testTriggers.go (hunk 1 above)
Comment (lsrinivas-pure) [reply tests/testTriggers.go]: removed it

File: tests/testTriggers.go (hunk 4 above)
Comment (lsrinivas-pure) [reply tests/testTriggers.go]: good catch. fixed it.

File: tests/testTriggers.go (hunk 2 above)
Comment (lsrinivas-pure) [reply tests/testTriggers.go:1243]: fixed

**Review by aravindranath14 — APPROVED** [review]
Looks good

**Pull request merged**